helm/botkube/values.yaml

```
@@ -8,7 +8,7 @@ extraAnnotations: {}
 # Priority class name for the pod
 priorityClassName: ""
 image:
-  repository: infracloudio/botkube
+  repository: ghcr.io/infracloudio/botkube
```

Let's separate out registry and repository fields and built the image name accordingly
```suggestion
  registry: ghcr.io
  repository: infracloudio/botkube
```

---

changes done